scripts/index.js

```diff
@@ -1,8 +1,9 @@
 const schedule = require('node-schedule');
+const { getJobs15Mins } = require('./parser');
```

Comment:
let's keep the import statement consistent, we will be using the format of:
```const parser = require('./parser');```
and when you need to call the function, just do `parser.getJobs15Mins()`